### ISSUE TITLE
Development and validation of the Hocus Focus Magic Performance Evaluation Scale for health professions personnel

### ISSUE THREAD

**Issue by ben-domingue** [ben-domingue/irw#536]
https://dataverse.harvard.edu/dataset.xhtml?persistentId=doi:10.7910/DVN/2DHW1G

**Comment by saviranadela** [comment]
working on this

**Comment by saviranadela** [comment]
paper: https://www.jeehp.org/upload/pdf/jeehp-16-08.pdf

@ben-domingue hi Ben! i have a question regarding the study described in the paper. the study mentions that two magician raters (Kevin Spencer and Max Darwin) evaluated participants' performances of three magic tricks using the 5-item HFMPES scale. however, the paper doesn’t provide much detail on how the raw data columns were labeled or what specific items they correspond to. based on the raw data, my best guess is that columns like KK, KPC, MPC, KRRB, and MK might represent the magicians, but the paper specifies that only two raters were involved.

_Phase II evaluated the psychometric properties of the HFMPES. This process involved 2 magicians using the HFMPES to independently evaluate 73 occupational therapy graduate students demonstrating 3 magic tricks. (a 4-point scale: 1=not relevant, 2=unable to assess relevance without major revision, 3=relevant but needs a minor alteration, and 4=very relevant and succinct.)_

i would appreciate your thoughts on this, as i’m unsure how to interpret the raw data columns in this context.

**Comment by ben-domingue** [comment]
ah this is great! ok, here is how i would set this up:
- id: the person doing the trick
- item: the magic trick (so there are 3 items here)
- rater: the magician doing the rating (two of them). 
- resp: the rating given by the magician to the person doing a specific trick

note the `rater` bit. this is a critical wrinkle here with this data and more rated data is great. but i can't tell what is going on with the columns either. i'm almost inclined to email them.

**Comment by saviranadela** [comment]
the columns still confuse me though, I think I should email them. what do you think?

**Comment by ben-domingue** [comment]
@saviranadela yeah that's great. do you want to email them and cc bdomingue@stanford.edu and itemresponsewarehouse@stanford.edu
